ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

```diff
@@ -240,3 +240,9 @@ oc exec -it  pod/mariadb-openstack -- mysql -uroot -p${MARIADB_PASSWD} -e "show
 # Post tests for keystone-operator
 # Check to confirm you can issue a token.
 openstack token issue
+
+# NOTE: if manila is deployed, build a default share required by tempest
+if [[ "${SERVICE_NAME}" == "MANILA" ]]; then
```

Comment:
I would say that the best place is in tempest step[1]. But tempest container doesn't have openstack client installed [2]. We can update the container if needed. Or we can just merge this here.
Thoughts?

[1] https://github.com/openshift/release/blob/master/ci-operator/step-registry/openstack-k8s-operators/tempest/openstack-k8s-operators-tempest-commands.sh
[2] https://github.com/openstack-k8s-operators/openstack-k8s-operators-ci/blob/master/images/tempest-container/Dockerfile

Reply:
hi! agree that adding this here is like a simple post test/config for manila-api.. it's good for now. if we ever have to have a more elaborate default share type (i suspect we won't need to), we can move the concern to tempest bootstrapping..

Reply:
@viroel ack and thanks for clarifying that aspect. My previous comment was made under different considerations, but I think one of the most important things is keeping our job steps as `individual` and `reusable`. Being that said, I updated the PR and  moved that piece of code in the tempest step.
Let me know if it looks good or requires more iterations.